lib/absinthe/phase/document/execution/resolution.ex

```diff
@@ -143,6 +143,17 @@ defmodule Absinthe.Phase.Document.Execution.Resolution do
     |> to_result(bp_field, full_type)
     |> walk_result(acc, bp_field, full_type, info)
   end
+  defp build_result({:error, %Error{} = err}, acc, bp_field, info, _) do
```

Comment:
@sylane: yes, precisely. If you're going to add extra stuff, explicitly call out the :message, maintain the same tuple size.